cli_tutorials/cli_charge_molecules.md

```diff
@@ -0,0 +1,103 @@
+# Generating Partial Charges with the OpenFE CLI
+
+This tutorial will show you how to use the OpenFE CLI to generate and store partial charges for a series of ligands 
+which can be used with OpenFE protocols. It is recommended to use a single set of charges for each ligand to ensure reproducibility
+between repeats or consistent charges between different legs of a calculation involving the same ligand, like a relative binding
+affinity calculation for example. As such both the `plan-rbfe-network` and `plan-rhfe-network` commands will calculate 
+partial charges for ligands making it expensive to run multiple network mappings while finding the optimal one for the resources 
+available. Hence, the `charge-molecules` command offers a way to generate and store the charges into an SDF file which 
+can be used with the rest of the OpenFE CLI and python API. 
+
+## Charging Molecules
+
+The `charge-molecules` command allows you to generate partial charges a series of small molecules saved in SDF or MOL2 
+format using the `am1bcc` method calculated using `ambertools`:
+
+```bash
+openfe charge-molecules -M tyk2_ligands.sdf -o charged_tyk2_ligands.sdf
+```
+
+This will result in a new SDF file `charged_tyk2_ligands.sdf` which contains the same ligands and their partial charges 
+stored in a new SD tag like so:
+
+```text
+lig_ejm_42
+     RDKit          3D
+
+ 35 36  0  0  0  0  0  0  0  0999 V2000
+   -4.7651   -2.8327  -16.5085 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -5.3566   -3.6931  -16.2274 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -4.7703   -4.9699  -16.2000 C   0  0  0  0  0  0  0  0  0  0  0  0
+[continues]
+ 28 31  1  0
+M  END
+
+>  <ofe-name>
+lig_ejm_42
+
+>  <atom.dprop.PartialCharge>
+0.14794282857142857 -0.096057171428571425 -0.12905717142857143 ...
+[continues]
+```
+
+Generating partial charges with the `am1bcc` method can be slow as they require a semi-empirical quantum chemical calculation, 
+we can however take advantage of multiprocessing to calculate the charges in parallel for each ligand which offers a 
+significant speed-up. The number of processors available for the workflow can be specified using the `-n` flag:
```

Comment:
I'd probably go as far as spell it out to readers and go "For example to spread out the calculation over 4 cores:"